README.md

```diff
@@ -16,7 +16,7 @@ Fluent UI Apple contains native UIKit and AppKit controls aligned with [Microsof
 #### Requirements
 
 - iOS 13+ or macOS 10.14+
-- Xcode 11+
+- Xcode 12.5+
```

Comment:
while our pipeline is updating... it doesn't mean the sdk requires 12.5 right?

Reply:
I agree that it's not a requirement that we use Xcode 12.5, but we also don't validate previous versions once we're running on 12.5.

I can revert it or put the Xcode version in a "recommended" section instead?

Reply:
I think having requires 11 makes sense for iOS 13+ requirement. yeah, maybe recommended latest Xcode might be something we could add?